examples/one-rate-plan/main.tf

```diff
@@ -1,3 +1,11 @@
+#############################################################################
+#local variables being used as hard coded values only for this example.
+#############################################################################
+
+locals {
+  access_tags = []
```

Comment:
`access_tags` should be put back as a variable, as its something we actually pass in the tests when executing the example